test-files/loadFromStorage_testdata/test1.json

```diff
@@ -1 +1 @@
tapoint":6900,"normalizedTime":1568705489,"timestamp":"2019-09-17T13:00:20.449997543+05:30"},{"datapoint":7000,"normalizedTime":1568705490,"timestamp":"2019-09-17T13:00:20.450004675+05:30"},{"datapoint":7100,"normalizedTime":1568705491,"timestamp":"2019-09-17T13:00:20.450011684+05:30"},{"datapoint":7200,"normalizedTime":1568705492,"timestamp":"2019-09-17T13:00:20.450018621+05:30"},{"datapoint":7300,"normalizedTime":1568705493,"timestamp":"2019-09-17T13:00:20.450025735+05:30"},{"datapoint":7400,"normalizedTime":1568705494,"timestamp":"2019-09-17T13:00:20.450041756+05:30"},{"datapoint":7500,"normalizedTime":1568705495,"timestamp":"2019-09-17T13:00:20.45004881+05:30"},{"datapoint":7600,"normalizedTime":1568705496,"timestamp":"2019-09-17T13:00:20.450056421+05:30"},{"datapoint":7700,"normalizedTime":1568705497,"timestamp":"2019-09-17T13:00:20.450065972+05:30"},{"datapoint":7800,"normalizedTime":1568705498,"timestamp":"2019-09-17T13:00:20.450075004+05:30"},{"datapoint":7900,"normalizedTime":1568705499,"timestamp":"2019-09-17T13:00:20.450085233+05:30"},{"datapoint":8000,"normalizedTime":1568705500,"timestamp":"2019-09-17T13:00:20.450092668+05:30"},{"datapoint":8100,"normalizedTime":1568705501,"timestamp":"2019-09-17T13:00:20.45009975+05:30"},{"datapoint":8200,"normalizedTime":1568705502,"timestamp":"2019-09-17T13:00:20.450106707+05:30"},{"datapoint":8300,"normalizedTime":1568705503,"timestamp":"2019-09-17T13:00:20.450113763+05:30"},{"datapoint":8400,"normalizedTime":1568705504,"timestamp":"2019-09-17T13:00:20.450120691+05:30"},{"datapoint":8500,"normalizedTime":1568705505,"timestamp":"2019-09-17T13:00:20.450127702+05:30"},{"datapoint":8600,"normalizedTime":1568705506,"timestamp":"2019-09-17T13:00:20.450135312+05:30"},{"datapoint":8700,"normalizedTime":1568705507,"timestamp":"2019-09-17T13:00:20.450146794+05:30"},{"datapoint":8800,"normalizedTime":1568705508,"timestamp":"2019-09-17T13:00:20.450154523+05:30"},{"datapoint":8900,"normalizedTime":1568705509,"timestamp":"2019-09-17T13:00:20.450161588+05:30"},{"datapoint":9000,"normalizedTime":1568705510,"timestamp":"2019-09-17T13:00:20.450168603+05:30"},{"datapoint":9100,"normalizedTime":1568705511,"timestamp":"2019-09-17T13:00:20.45017554+05:30"},{"datapoint":9200,"normalizedTime":1568705512,"timestamp":"2019-09-17T13:00:20.450187867+05:30"},{"datapoint":9300,"normalizedTime":1568705513,"timestamp":"2019-09-17T13:00:20.450199331+05:30"},{"datapoint":9400,"normalizedTime":1568705514,"timestamp":"2019-09-17T13:00:20.450210255+05:30"},{"datapoint":9500,"normalizedTime":1568705515,"timestamp":"2019-09-17T13:00:20.450218366+05:30"},{"datapoint":9600,"normalizedTime":1568705516,"timestamp":"2019-09-17T13:00:20.450229149+05:30"},{"datapoint":9700,"normalizedTime":1568705517,"timestamp":"2019-09-17T13:00:20.450237776+05:30"},{"datapoint":9800,"normalizedTime":1568705518,"timestamp":"2019-09-17T13:00:20.450245113+05:30"},{"datapoint":9900,"normalizedTime":1568705519,"timestamp":"2019-09-17T13:00:20.450252257+05:30"}]
```

Comment:
This should not be changed. Kindly add the previous value. Thanks

Reply:
I changed the normalizedTime, which was same for all the blocks. Traversal wouldn't work other wise.

Reply:
okay sure